<ui version="4.0" >
 <class>WhiteBalanceFormClass</class>
 <widget class="QDialog" name="WhiteBalanceFormClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WhiteBalanceForm</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="12" >
    <widget class="QDockWidget" name="dwRGBGraph" >
     <property name="features" >
      <set>QDockWidget::NoDockWidgetFeatures</set>
     </property>
     <widget class="QWidget" name="dockWidgetContents" />
    </widget>
   </item>
   <item rowspan="2" row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Red</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbRedAdjustNumb" />
     </item>
     <item>
      <widget class="QLabel" name="lbRedRange" />
     </item>
    </layout>
   </item>
   <item rowspan="2" row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="3" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Green</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbGreenAdjustNumb" />
     </item>
     <item>
      <widget class="QLabel" name="lbGreenRange" />
     </item>
    </layout>
   </item>
   <item rowspan="2" row="1" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="5" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbBlueAdjustNumb" />
     </item>
     <item>
      <widget class="QLabel" name="lbBlueRange" />
     </item>
    </layout>
   </item>
   <item rowspan="2" row="1" column="6" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7" >
    <widget class="QSpinBox" name="sbShading" >
     <property name="maximum" >
      <number>255</number>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="8" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>81</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="9" >
    <widget class="QPushButton" name="pbSave" >
     <property name="minimumSize" >
      <size>
       <width>101</width>
       <height>51</height>
      </size>
     </property>
     <property name="text" >
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="10" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>51</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="1" column="11" >
    <widget class="QPushButton" name="pbClose" >
     <property name="minimumSize" >
      <size>
       <width>101</width>
       <height>51</height>
      </size>
     </property>
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7" >
    <widget class="QPushButton" name="pbShading" >
     <property name="minimumSize" >
      <size>
       <width>50</width>
       <height>29</height>
      </size>
     </property>
     <property name="text" >
      <string>Shading</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
